packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -195,6 +195,9 @@ export const tableRowBorderColor = DesignToken.create<string>(
 export const controlHeight = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.controlHeight)
 ).withDefault('32px');
+export const slimControlHeight = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.slimControlHeight)
+).withDefault('24px');
```

Comment:
This doesn't seem to align with out token naming pattern: https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#token-naming

`[element]-[part]-[state]-[token_type]`

I'd expect the abstract "element" to be `control`
The part is not defined
The "state" is 'slim'
The type is 'height'

So `controlSlimHeight`, if you have any Qs about the pattern we should discuss it / clarify the docs 👍

Reply:
I thought the `element` could have an adjective preceeding it. I think most of the instances of this I saw were actually following the `element-part` pattern, e.g. `buttonLabelBlahBlah`. There are still `errorTextBlahBlah` tokens, and I think those may just be violating the pattern. Anyway, I will change the name.